sources/packages/backend/libs/test-utils/src/factories/education-program.ts

```diff
@@ -8,10 +8,13 @@ import {
 } from "@sims/sims-db";
 import { createFakeInstitution } from "@sims/test-utils";
 
-export function createFakeEducationProgram(relations?: {
-  auditUser: User;
-  institution?: Institution;
-}): EducationProgram {
+export function createFakeEducationProgram(
+  relations?: {
+    auditUser: User;
+    institution?: Institution;
+  },
+  options?: { programInitialValues?: Partial<EducationProgram> },
```

Comment:
here it can be `initalValues` instead of `programInitialValues` as Program is the entity here